src/config/model.rs

```diff
@@ -367,16 +353,60 @@ impl Config {
     /// This is a little helper-function like which uses the the name and email
     /// of the account to create a valid address for the header of the envelope
     /// of an msg.
+    ///
+    /// # Hint
+    /// If the name includes some special characters like a whitespace, comma or semicolon, then
+    /// the name will be automatically wrapped between two `"`.
+    /// 
+    /// # Exapmle
+    /// ```
+    /// use himalaya::config::model::{Account, Config};
+    /// 
+    /// fn main() {
+    ///     let config = Config::default();
+    ///
+    ///     let normal_account = Account::new(Some("Acc1"), "acc1@mail.com");
+    ///     // notice the semicolon in the name!
+    ///     let special_account = Account::new(Some("TL;DR"), "acc2@mail.com");
+    ///
+    ///     // -- Expeced outputs --
+    ///     let expected_normal = Account {
+    ///         name: Some("Acc1".to_string()),
+    ///         email: "acc1@mail.com".to_string(),
+    ///         .. Account::default()
+    ///     };
+    ///
+    ///     let expected_special = Account {
+    ///         name: Some("\"TL;DR\"".to_string()),
+    ///         email: "acc2@mail.com".to_string(),
+    ///         .. Account::default()
+    ///     };
+    ///
+    ///     assert_eq!(config.address(&normal_account), "Acc1 <acc1@mail.com>");
+    ///     assert_eq!(config.address(&special_account), "\"TL;DR\" <acc2@mail.com>");
+    /// }
+    /// ```
     pub fn address(&self, account: &Account) -> String {
         let name = account.name.as_ref().unwrap_or(&self.name);
+
+        let special_chars: Vec<&str> = name
+            .matches(|character| ",.:;".contains(character))
+            .collect();
+
         if name.is_empty() {
             format!("{}", account.email)
+        } else if !special_chars.is_empty() {
+            // so the name has special characters => Wrap it with '"'
+            format!("\"{}\" <{}>", name, account.email)
```

Comment:
I just try to find a good compromise between value of feature and time spent, we see how complex and full of details those RFC are. Putting quotes avoid invalid names. Putting quotes around a name that does not need them seems not to make it invalid either. Knowing that, do you think it's worth spending time digging the RFC to have a perfect working model (with tests) rather than just putting quotes ? I'm open to discussion ofc, I may be wrong there, it is just my opinion.

Reply:
So you'd prefer to have always quotes?

Reply:
As you wish. I don't want to force anything. After all, if it's just chars to check then it's okey, you can leave this way.